### PR TITLE
[MOB-2991] Add Verify Quotes endpoint

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-2991](https://karhoo.atlassian.net/browse/MOB-2991)

## DESCRIPTION
Added verify quotes endpoint, created interactor and add tests.

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [x] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        runBlocking {
+            interactor.execute { result ->
+                when (result) {
+                    is Resource.Success -> returnedVerifyInfo = result.data
+                    is Resource.Failure -> Assert.fail()
+                }
+            }
+            delay(5)
+        }
+        Assert.assertNotNull(returnedVerifyInfo)
```
Comment (jsm20) [inline sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt:60]: Use static import for these

File: sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            }
+            delay(5)
+        }
+
+        Assert.assertEquals(KarhooError.InternalSDKError, error)
+        Assert.assertNull(shouldBeNull)
+    }
+
+    companion object {
+        val quoteType = QuoteType.ESTIMATED
```
Comment (jsm20) [inline sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt:117]: Should probably be const and names in caps

Comment (EdwardW2) [reply sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt:117]: Not sure if it is required to be in Caps and const does not work with this so far. Android does not like the usage of Const on these val's. 

But should be fine to be as it is, taken from another companion object in other interactor tests so doesn't seem to be an issue.

Comment (jsm20) [reply sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt:117]: https://kotlinlang.org/docs/reference/coding-conventions.html

**Review by jsm20 — APPROVED** [review]

**Pull request merged**